src/NodeDev.Blazor/Index.razor

```diff
@@ -104,14 +99,17 @@
 
     protected override void OnInitialized()
     {
-        if (File.Exists("project.json"))
-            Project = Core.Project.Deserialize(File.ReadAllText("project.json"));
-        else
-            Project = Core.Project.CreateNewDefaultProject();
+        ProjectService.ProjectChanged += OnProjectChanged;
 
-        DebuggedPathService.ChangeProject(Project);
+        DebuggedPathService.ChangeProject(ProjectService.Project);
     }
 
+    private void OnProjectChanged()
+	{
+        DebuggedPathService.ChangeProject(ProjectService.Project);
+        NavigationManager.Refresh(true);
```

Comment:
We'll need to wrap that in a `Invoke` call. It works when there's only 1 tab opened but if there's more than one It'll access the UI from the wrong thread. I think it will throw an exception if we're currently rendering when the project is changed in another tab